vscode-cedar-wasm/Cargo.toml

```diff
@@ -44,3 +44,6 @@ cargo-lock = "8.0.2"
 [profile.release]
 # Tell `rustc` to optimize for small code size.
 opt-level = "s"
+
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = false
```

Comment:
intentionally checked in or just for debugging?

Reply:
Good catch.  I was getting a (new) local error so I added.  Might be result of changes on my system from testing including DWARF for wasm debugging.

```
[INFO]: Optimizing wasm binaries with `wasm-opt`...
[parse exception: expected more elements in list]
Fatal: error parsing wasm
Error: failed to execute `wasm-opt`: exited with exit status: 1
```

looks like because of https://github.com/WebAssembly/binaryen/issues/5926